curriculum/challenges/_meta/responsive-web-design-projects/meta.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "Responsive Web Design Projects",
   "isUpcomingChange": false,
+  "usesMultifileEditor": true,
```

Comment:
Can this go into `main` or does it need to wait for when we release the multifile projects?

Reply:
nice catch. Technically, it could go into main. They currently use the project view so the editor would never show up for it to matter. I removed it anyway for now.